test/gitpod.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as logging from '../src/logging';
+import { Project } from '../src/project';
+import { synthSnapshot } from './util';
+
+// This is duplicated vs exported
+const GITPOD_FILE = '.gitpod.yml';
+
+logging.disable();
+
+let tempDir: string;
+beforeEach(() => {
+  tempDir = fs.mkdtempSync(path.join(__dirname, 'tmp.gitpod'));
+});
+
+afterEach(() => {
+  if (tempDir) {
+    fs.removeSync(tempDir);
+  }
+});
+
+describe('gitpod enable/disable', () => {
+  test('given gitpod is false', () => {
+    const project = new Project({
+      outdir: tempDir,
+      gitpod: false,
+    });
+
+    project.synth();
+    const filePath = path.join(tempDir, GITPOD_FILE);
+    expect(fs.existsSync(filePath)).toBeFalsy();
+  });
+  test('given gitpod is true', () => {
+    const project = new Project({
+      outdir: tempDir,
+      gitpod: true,
+    });
+
+    project.synth();
+    const filePath = path.join(tempDir, GITPOD_FILE);
+    expect(fs.existsSync(filePath)).toBeTruthy();
+
+    const snapshot = synthSnapshot(project)[GITPOD_FILE];
+    expect(snapshot).toContain('tasks:');
+    expect(snapshot).toContain('command: echo Initialized');
+  });
+});
+
+describe('gitpod image & file', () => {
+  test('given an image', () => {
+    const project = new Project({
+      outdir: tempDir,
+      gitpod: true,
+    });
+    if (project.gitpod) {
+      project.gitpod.addCustomDocker({ image: 'jsii/superchain' });
+    }
```

Comment:
```suggestion
    project.gitpod?.addCustomDocker({ image: 'jsii/superchain' });
```
Same change can be made for the other tests below.

Reply:
oh yeah duh.